package.json

```diff
@@ -712,6 +718,10 @@
                     "command": "aws.s3.uploadFile",
                     "when": "false"
                 },
+                {
+                    "command": "aws.s3.uploadFilePalette",
+                    "when": "true"
+                },
```

Comment:
Nit: I think `when: true` is not necessary. You should be able to remove this from here; commands should appear in the command palette unless told otherwise.